students/jaeyeon-won/project_westagram/accounts/views.py

```diff
@@ -0,0 +1,83 @@
+import json
+import re
+
+from django.views import View
+from django.http  import JsonResponse
+
+from .models      import User
+
+class SignupView(View):
+    def post(self,request):
+        data = json.loads(request.body)
+
+        try:
+            name                = data['name']
+            phone_number        = data['phone_number']
+            email               = data['email']
+            username            = data ['username']
+            password            = data['password']
+            email_vaildation    = re.match('[a-zA-Z0-9._+-]+@[a-z0-9-]+\.[a-z.]+',email)
+            password_vaildation = re.match('^(?=.*[a-zA-Z0-9.,-]).{8,}$',password)
+            
+            if name =='':
+                return JsonResponse({'message' : '이름을 입력하세요!'}, status = 400)
```

Comment:
리턴 메시지는 클라이언트가 아닌 프론트에게 주는 메시지 입니다. 아래와 같이 영어로 간단하게 보내면 좋을 것 같습니다.
```suggestion
                return JsonResponse({'message' : 'message":"CHECK_YOUR_INPUT'}, status = 400)
```